pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -697,6 +697,16 @@ in
     buildFlags = [ "--disable-lto" ];
   });
 
+  sass-embedded = attrs: {
```

Comment:
If `attrs` is not being used inside the function, the convention on the functional gang is to use underscore, `_`.